server/snapshot/snapshot.go

```diff
@@ -96,14 +102,11 @@ func contentExpired(gun string, sn *data.SignedSnapshot, store storage.MetaStore
 		if err != nil {
 			return false
 		}
-		roleExp, newHash := roleExpired(curr, meta)
+		roleExp, newMeta := roleExpired(curr, meta)
 		if roleExp {
-			updatedMeta[role] = data.FileMeta{
-				Length: int64(len(curr)),
-				Hashes: data.Hashes{
-					"sha256": newHash,
-				},
-			}
+			updatedMeta[role] = newMeta
+		} else {
+			updatedMeta[role] = meta
```

Comment:
Could we add a test for this case, where some of the content has expired and other content has not?